cmd/aws/aws.go

```diff
@@ -83,43 +83,50 @@ func getAwsCredentials(sess *session.Session) *credentials.Credentials {
 	return creds
 }
 
-func getAwsSession() *session.Session {
+func getAwsSession() (*session.Session, error) {
 	if Session == nil {
 		var err error
 		Session, err = session.NewSessionWithOptions(session.Options{
 			SharedConfigState: session.SharedConfigEnable,
 		})
-		common.HandleError(err)
+		if err != nil {
+			return nil, err
+		}
 	}
-	return Session
+	return Session, nil
 }
 
-func getAwsSecretsManagerClient() *secretsmanager.SecretsManager {
+func getAwsSecretsManagerClient() (*secretsmanager.SecretsManager, error) {
 	if SecretsManagerClient == nil {
+		sess, err := getAwsSession()
+		if err != nil {
+			return nil, err
+		}
 		SecretsManagerClient = secretsmanager.New(
-			getAwsSession(),
+			sess,
 			aws.NewConfig().WithRegion(Region).WithCredentials(
-				getAwsCredentials(getAwsSession())))
+				getAwsCredentials(sess)))
 	}
-	return SecretsManagerClient
+	return SecretsManagerClient, nil
 }
 
-func GetAwsSecret(secretName string, secretKey string) string {
+func GetAwsSecret(secretName string, secretKey string) (string, error) {
+
+	var secretStr = ""
```

Comment:
Is it necessary to declare `secretStr` here in the outer scope?  It already uses `:=` assignments anywhere it is used

Reply:
Nope... I'll delete it - I think I was planning to do something different there but ended up not needing it